src/AstGen.zig

```diff
@@ -242,6 +255,9 @@ pub const ResultLoc = union(enum) {
     /// is inferred based on peer type resolution for a `Zir.Inst.Block`.
     /// The result instruction from the expression must be ignored.
     block_ptr: *GenZir,
+    /// Use in then-scope of loop, only work for 'break' instruction,
+    /// otherwise it's work like none.
```

Comment:
I don't understand what this does based on this comment. And why is it only in the then-scope?